public/portfolio_shared_data.json

```diff
@@ -5,13 +5,13 @@
       "social": [
         {
           "name": "github",
-          "url": "https://github.com",
+          "url": "https://github.com/DavidMatalik",
           "class": "fab fa-github"
         },
         {
-          "name": "instagram",
-          "url": "https://www.instagram.com",
-          "class": "fab fa-instagram"
+          "name": "linkedin",
+          "url": "https://www.linkedin.com/",
```

Comment:
I guess this will be finalized later

Reply:
Yes exactly, when there is a linkedIn Profile